docs/schema/keywordinclusion-simpletype-sps15xsdsearchset5.md

```diff
@@ -18,11 +18,11 @@ description: "Last modified: March 09, 2015"
   
 ## Type information
 
-|||
+| Resource | File |
 |:-----|:-----|
-|**Base type** <br/> |xs:string  <br/> |
-|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Query  <br/> |
-|**Schema file** <br/> |schema_Microsoft.Office.Server.Search.Query.xsd  <br/> |
+|Base type <br/> |xs:string  <br/> |
```

Comment:
Same here, leave in the asterisks for this table, which bold the items in the first column.  You can however delete all the HTLM tags for breaks - <br/>